OpenStack-accounting/sql/cinder_get_accounting_data.sql

```diff
@@ -0,0 +1,44 @@
+CREATE PROCEDURE `get_accounting_data`(IN starttime datetime, IN endtime datetime)
+BEGIN
+/*
+   This procedure generates accounting data for cinder
+*/
+SELECT
+    IFNULL(v.availability_zone, 'nova') AS AvailabilityZone,
```

Comment:
Just to double check, here is it only selecting `nova` as the availability zone if there isn't an availability zone for a given volume (e.g. volume doesn't have a ceph availability zone and is just `null`)?

Reply:
Yes, that is what that's doing